Crashlytics/Crashlytics/Rollouts/CrashlyticsRemoteConfigManager.swift

```diff
@@ -35,6 +35,22 @@ public class CrashlyticsRemoteConfigManager: NSObject {
   @objc public func updateRolloutsState(rolloutsState: RolloutsState) {
     rolloutAssignment = normalizeRolloutAssignment(assignments: Array(rolloutsState.assignments))
   }
+
+  @objc public func getRolloutAssignmentsEncodedJson() -> String? {
+    let encodedRolloutAssignments = rolloutAssignment.map { assignment in
+      EncodedRolloutAssignment(assignment: assignment)
+    }
+
+    let encoder = JSONEncoder()
+    encoder.keyEncodingStrategy = .convertToSnakeCase
+    encoder.outputFormatting = .sortedKeys
+    let encodeData = try? encoder.encode(encodedRolloutAssignments)
+    if let data = encodeData, let returnString = String(data: data, encoding: .utf8) {
+      return returnString
+    }
+    debugPrint("Failed to serialize rollouts", encodeData ?? "nil")
```

Comment:
I think we'll want to hook into logging functions in the future. Should we add a TODO here?

Reply:
Ah you are right, for sure!

Reply:
Done